### PR TITLE
Add stream option to text generation task

### PR DESCRIPTION
**Pull request by jonatanklosko**
https://github.com/livebook-dev/kino_bumblebee/assets/17034772/3f2e6adc-bff3-4023-9b97-b1315cfd451b

### CODE REVIEW

File: lib/kino_bumblebee/task_cell.ex [hunk 1]
```diff
@@ -844,6 +845,7 @@ defmodule KinoBumblebee.TaskCell do
   end
 
   def handle_event("update_field", %{"field" => field, "value" => value}, ctx) do
+    IO.inspect({field, value})
```
Comment (josevalim) [inline lib/kino_bumblebee/task_cell.ex:848]: :D

Comment (jonatanklosko) [reply lib/kino_bumblebee/task_cell.ex]: 🙈

**Review by josevalim — COMMENTED** [review]
I think we should always stream, no option. Thoughts?

**Comment by jonatanklosko** [comment]
They may want to do either in an actual application, so this way we have an example of both. I don't have a strong preference though, your call!

**Comment by josevalim** [comment]
I would go with the simpler UI and simpler code. They can remove the option if they feel adventurous. :)

File: lib/assets/task_cell/main.js [hunk 2]
```diff
@@ -126,8 +126,52 @@ export async function init(ctx, payload) {
     `,
   };
 
+  const BaseSwitch = {
```
Comment (jonatanklosko) [inline lib/assets/task_cell/main.js:129]: I'm gonna keep this, since we will use it in the future. In particular for #18.

**Review by josevalim — APPROVED** [review]

**Pull request merged**